frontend/public/components/service-account.jsx

```diff
@@ -20,7 +20,7 @@ const KubeConfigify = (kind, sa) => ({
 
     k8sGet(SecretModel, name, namespace).then(({data}) => {
       const server = window.SERVER_FLAGS.kubeAPIServerURL;
-      const clusterName = window.SERVER_FLAGS.clusterName;
+      const clusterName = window.SERVER_FLAGS.kubeAPIServerURL.replace(/(^\w+:|^)\/\//, '');
```

Comment:
I'd probably use the `URL` utility instead of a regex. It's required already by the console.

```suggestion
      const url = new URL(server);
      const clusterName = url.port ? `${url.hostname}:{url.port}` : url.hostname;
```